src/io/flutter/FlutterInitializer.java

```diff
@@ -110,10 +110,18 @@ public void runActivity(@NotNull Project project) {
     // Convert all modules of deprecated type FlutterModuleType.
     if (FlutterModuleUtils.convertFromDeprecatedModuleType(project)) {
```

Comment:
I think that as-is, all WEB_MODULE projects are going to be converted to JAVA_MODULE projects, regardless if the project is Flutter.

Reply:
oops!

I'll change this to == WEB_MODULE and has a dependency on flutter in the pubspec. I think that should be a good heuristic.

Reply:
(done)